sema/discovery/discovery.py

```
@@ -127,35 +127,52 @@ def _make_response(
+            "json-ld",
+            "n3",
+            "nt",
+            "trig",
+            "nquads",
+            "xml",
+        ]
+
+        for fmt in formats_to_try:
+            try:
+                g: Graph = Graph().parse(
+                    data=content, format=fmt, publicID=source_url
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)